src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,6 +1,11 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
+
+
+
```

Comment:
Let’s remove redundant empty lines

FYI: where we use an empty string:
1) for the block of imports;
2) at the end of the file. the file must end with one blank line
3) to separate fields, constructors, static and non-static initialization blocks, and methods (but do not leave a blank line in front of class fields).
4) within the method for separating complex logic (extremely rare)
In other cases, an empty line is unnecessary!